services/online_exam_service/src/routes/student.quiz.routes.js

```
@@ -5,12 +5,19 @@ const router = express.Router();
 const studentQuizController = require('../controllers/student.quiz.controller');
 const { authenticateToken, checkPermission } = require('../middleware/auth');
 
+const { authenticateToken } = require('../middleware/auth.middleware');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'authenticateToken'. Consider to delete it or rename it.

'authenticateToken' is defined here:


(lint/suspicious/noRedeclare)